clarify document failure reasons; proof of address info; front of id … (#619)
Dwolla/dev-portal: pages/docs/balance/business-verified-customer/handle-verification-statuses.mdx

| ------------------------- | ---------------------------------------------------------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
| `BusinessDocNotSupported` | Business document not supported                                        | The business document provided is not supported for verification. Please request approved [business documentation](#document-types) from the end user for verification.                                                                                                                                              |

> | `BusinessNameMismatch`    | Business name on account does not match document                       | The legal business name listed in the documentation uploaded does not match the Registered Business Name on the account. The account has been moved to [retry](#handling-status-retry) so that the business name listed at the account level can be adjusted to match the business documentation which was uploaded. |

Reviewer: ```suggestion
| `BusinessNameMismatch`    | Business name on account does not match document                       | The legal business name listed in the documentation uploaded does not match the Registered Business Name on the account. The account has been moved to [retry](#handling-retry-status) so that the business name listed at the account level can be adjusted to match the business documentation which was uploaded. |
```

> | `BusinessTypeMismatch`    | Business type chosen does not match document                           | Based on the documentation provided, the entity type for this business should be created as a/an (LLC, Corporation, Sole Prop). The account has been moved to [retry](#handling-status-retry) so that the correct business type (LLC, Corporation, Sole Prop etc.) can be submitted.                                 |

Reviewer: ```suggestion
| `BusinessTypeMismatch`    | Business type chosen does not match document                           | Based on the documentation provided, the entity type for this business should be created as a/an (LLC, Corporation, Sole Prop). The account has been moved to [retry](#handling-retry-status) so that the correct business type (LLC, Corporation, Sole Prop etc.) can be submitted.                                 |
```

> | `ScanDobMismatch`         | Scan DOB does not match DOB on account                                 | The DOB listed on the ID uploaded does not match the DOB on the user's account. The account has been placed in [retry](#handling-status-retry) so that the user can adjust the DOB listed on the account to match the ID which was provided.                                                                         |

Reviewer: ```suggestion
| `ScanDobMismatch`         | Scan DOB does not match DOB on account                                 | The DOB listed on the ID uploaded does not match the DOB on the user's account. The account has been placed in [retry](#handling-retry-status) so that the user can adjust the DOB listed on the account to match the ID which was provided.                                                                         |
```

> | `ScanFailedOther`         | ID may be fraudulent or a generic example ID image                     | The ID uploaded may be fraudulent or a generic example of an ID. The user needs to upload an valid ID to proceed with account verification.                                                                                                                                                                          |

Reviewer: ```suggestion
| `ScanFailedOther`         | ID may be fraudulent or a generic example ID image                     | The ID uploaded may be fraudulent or a generic example of an ID. The user needs to upload a valid ID to proceed with account verification.                                                                                                                                                                          |
```

| `ScanIdExpired`           | ID is expired or missing expiration date                               | The ID uploaded by the user is expired. The user will need to upload a non-expired ID.                                                                                                                                                                                                                               |

> | `ScanIdTypeNotSupported`  | ID may be a military ID, firearm license, or other unsupported ID type | The uploaded ID is not an acceptable form of ID. [Here](#acceptable-documents) is a list of ID types that Dwolla accepts for account verification.                                                                                                                                                                   |

Reviewer: ```suggestion
| `ScanIdTypeNotSupported`  | ID may be a military ID, firearm license, or other unsupported ID type | The uploaded ID is not an acceptable form of ID. [Here](#document-types) is a list of ID types that Dwolla accepts for account verification.                                                                                                                                                                   |
```

| `ScanIdUnrecognized`      | ID is not recognized                                                   | The ID which has been uploaded is unreadable. The user will need to upload a new image of their ID to proceed with verification.                                                                                                                                                                                     |

> | `ScanNameMismatch`        | Scan name does not match name on account                               | The name listed on the ID which has been uploaded by the user does not match the name which is listed at the account level. The account has been placed in [retry](#handling-status-retry) so that the user can adjust the name on the account to match the ID which was provided.                                   |

Reviewer: ```suggestion
| `ScanNameMismatch`        | Scan name does not match name on account                               | The name listed on the ID which has been uploaded by the user does not match the name which is listed at the account level. The account has been placed in [retry](#handling-retry-status) so that the user can adjust the name on the account to match the ID which was provided.                                   |
```

> | `ScanNotReadable`         | Image blurry, too dark, or obscured by glare                           | The uploaded ID is blurry, cutoff, or unreadable. The user will need to upload a clear, color, camera-captured image of their ID to proceed with verification. Here are some [best practices](#handling-status-document) related to document uploads.                                                                |

Reviewer: ```suggestion
| `ScanNotReadable`         | Image blurry, too dark, or obscured by glare                           | The uploaded ID is blurry, cutoff, or unreadable. The user will need to upload a clear, color, camera-captured image of their ID to proceed with verification. Here are some [best practices](#handling-document-status) related to document uploads.                                                                |
```

> | `ScanNotUploaded`         | Scan not uploaded                                                      | The uploaded image is not an ID. The user will need to upload an image of a valid ID to proceed. [Here](#acceptable-documents) is a list of valid ID's that Dwolla accepts for account verification.                                                                                                                 |

Reviewer: ```suggestion
| `ScanNotUploaded`         | Scan not uploaded                                                      | The uploaded image is not an ID. The user will need to upload an image of a valid ID to proceed. [Here](#document-types) is a list of valid ID's that Dwolla accepts for account verification.                                                                                                                 |
```
